atom_tools/lib/converter.py

```diff
@@ -506,12 +503,10 @@ def _create_param_object(self, ep: str, orig_ep: str, call: Dict | None) -> List
 
         """
         params = self.generic_params_helper(ep, orig_ep) if '{' in ep else []
-        if not params and call:
-            ptypes = set(call.get('paramTypes', []))
-            if len(ptypes) > 1:
-                params = [{'name': param, 'in': 'header'} for param in ptypes if param != 'ANY']
-            else:
-                params = [{'name': param, 'in': 'header'} for param in ptypes]
+        # if not params and call:
+        #     ptypes = set(call.get('paramTypes', []))
+        #     # FIXME: This is not a correct assumption. Perhaps the parameter types could be passed as an `x-` attribute
```

Comment:
@prabhu This is a placeholder. I am not assuming that it is actually the name. It needs improvement but it would be pointless to add to an x- attribute as by the time anyone builds support for that, I will have replaced with the correct parameter names. I will focus on getting the correct names after I conclude filtering - which I expect to finish tonight.

Reply:
hmm ok. happy to wait in case you are in the middle of something.